raiden/api/rest.py

```diff
@@ -148,6 +149,10 @@
         '/tokens/<hexaddress:token_address>',
         RegisterTokenResource,
     ),
+    (
+        '/in_flight',
```

Comment:
I have multiple comments about this:
1. Nickpick: I wouldn't call it `in_flight` but rather something like `pending_payments`, `pending_transfers` ... etc. But i would like someone else to also step in and weigh-in on the matter as it is a matter of taste.
2. This will return all pending transfers which assumes that we have a single page which shows them. However, we might want to consider being able to get pending transfers for a single channel.
3. Follow up to (2), most actions we have to list things are bound to the channel resource so it might be a good idea to follow the "standard" of whats already there. 
Opinions: @LefterisJP @hackaugusto ?

Reply:
So, my preference is to have a RESTful api, the problem is this design is already broken, e.g. `/payments/` endpoint receives a payment description for the `POST` but returns events for `GET`.

Reply:
no strong opinions here. But yeah I would also go for `pending_transfers` and not `in_flight`